tooling/ssa_fuzzer/src/builder.rs

```diff
@@ -0,0 +1,365 @@
+#![allow(dead_code)]
+use crate::compiler::compile;
+use crate::config::NUMBER_OF_VARIABLES_INITIAL;
+use crate::helpers;
+use acvm::FieldElement;
+use noirc_driver::{CompileOptions, CompiledProgram};
+use noirc_evaluator::ssa::function_builder::FunctionBuilder;
+use noirc_evaluator::ssa::ir::basic_block::BasicBlockId;
+use noirc_evaluator::ssa::ir::function::{Function, RuntimeType};
+use noirc_evaluator::ssa::ir::instruction::BinaryOp;
+use noirc_evaluator::ssa::ir::map::Id;
+use noirc_evaluator::ssa::ir::types::{NumericType, Type};
+use noirc_evaluator::ssa::ir::value::Value;
+use noirc_frontend::monomorphization::ast::InlineType as FrontendInlineType;
+use std::panic::AssertUnwindSafe;
+use std::sync::Arc;
+use thiserror::Error;
+
+#[derive(Debug, Error)]
+pub enum FuzzerBuilderError {
+    #[error("Compilation panicked")]
```

Comment:
```suggestion
    #[error("Compilation panicked: {0}")]
```
At least I think we have to include the field to actually see the value.